### PR TITLE
Update scala-fmt to version 2.4.4

### PR DESCRIPTION
**Pull request by paulo-coutinho**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -19,4 +19,3 @@ lazy val djinni = (project in file("."))
     assembly / assemblyOption := (assembly / assemblyOption).value.copy(prependShellScript = Some(defaultUniversalScript(shebang = false))),
     assembly / test := {}
   )
-  inConfig(IntegrationTest)(org.scalafmt.sbt.ScalafmtPlugin.scalafmtConfigSettings)
```
Comment (a4z) [inline build.sbt:22]: documentation says this is required
https://scalameta.org/scalafmt/docs/installation.html#enable-integrationtest
to also format the integration tests, and not only the source code

**Review by a4z — CHANGES_REQUESTED** [review]
Please update the PR title, this is not a fix but a version update

File: build.sbt (hunk 1 above)
Comment (paulo-coutinho) [reply build.sbt:22]: It is not necessary, because i open an issue. Maybe the docs are not updated.
https://github.com/scalameta/sbt-scalafmt/issues/152

Comment (jothepro) [reply build.sbt:22]: I can confirm that. Thanks, @paulo-coutinho !

**Review by jothepro — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (a4z) [reply build.sbt:22]: indeed

**Review by a4z — APPROVED** [review]
Thanks!

**Pull request merged**